rules/prefer-to-contain.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const getDocsUrl = require('./util').getDocsUrl;
+const expectCase = require('./util').expectCase;
+const expectNotCase = require('./util').expectNotCase;
+const expectResolveCase = require('./util').expectResolveCase;
+const expectRejectCase = require('./util').expectRejectCase;
+const method = require('./util').method;
+const argument = require('./util').argument;
+
+module.exports = {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
```

Comment:
this `if` is really verbose, could you clean it up? Not a bigge (we can do it later), but 15 lines seems excessive

Reply:
Broke it in multiple methods